api/__init__.py

```diff
@@ -0,0 +1,40 @@
+import requests
+
+DEFAULT_CONTENT_TYPE = "application/json"
+DEFAULT_ACCEPT = "application/vnd.ceph.api.v1.0+json"
```

Comment:
Since this is generic api package module, Go as per the user requests and make it independent.

This suggestion is because,Since we may have to work on prometheues, grafana and any other APIs in future.

Reply:
Get headers from the ceph Restful API module.

Reply:
+1, will have default header instead. which will be overridden while calling method